intel-sgx/ias/examples/attestation-tool.rs

```diff
@@ -152,44 +136,33 @@ async fn main() {
 
     match ias_client.verify_quote(quote.quote()).await {
         Ok(response) => {
-            let report: AttestationEmbeddedIasReport<Unverified> = response
-                .clone()
-                .into();
-            let report = report
-                .verify::<Mbedtls>(&[IAS_REPORT_SIGNING_CERTIFICATE.as_slice()])
-                .and_then(|rep| {
-                    rep
-                        .to_attestation_evidence_reponse()?
-                        .verify(&IgnorePlatformState::new())
-                });
-
-            let report = match report {
-                Ok(report) => report,
-                Err(err) => {
-                    println!("IAS verification error:");
-                    println!("  {}", err);
-                    return;
-                }
-            };
+            let report = response
+                .verify::<Mbedtls>(&[TEST_REPORT_SIGNING_CERT.as_slice()])
+                .expect("Corrupt report");
```

Comment:
Don't expect. Just print an error message and quit. Also, the error message should be "Unable to verify signature on IAS report".

Reply:
Fixed